demo/package.json

```diff
@@ -47,7 +47,6 @@
     "babel-preset-es2015": "^6.3.13",
     "babel-preset-react": "^6.3.13",
     "babel-preset-stage-2": "^6.3.13",
-    "brfs-babel": "^1.0.0",
```

Comment:
Do we still need the transform-loader after removing this (Per my understanding, the webpack transform loader is used to run browserify transforms under webpack, and brfs-babel is a browserify transform).